### DISCUSSION TITLE
Did the ABI change between 1.14.1 and 1.14.0?

### DISCUSSION THREAD

**Discussion by mkitti** [HDFGroup/hdf5#3132] [Q&A] [answered]
Did the application binary interface change between HDF5 1.14.0 and 1.14.1?

xref:
https://github.com/conda-forge/hdf5-feedstock/pull/202#issuecomment-1592236319

**Comment by lrknox** [comment, marked as answer]
Yes, 5 new C functions were added;  see https://confluence.hdfgroup.org/display/HDF5/Software+Changes+from+Release+to+Release+for+HDF5+1.14 or https://confluence.hdfgroup.org/display/support/HDF5%201.14.1#compatibility1141.


From: Mark ***@***.***>
Sent: Wednesday, June 14, 2023 11:18 PM
To: ***@***.***>
Cc: ***@***.***>
Subject: [HDFGroup/hdf5] Did the ABI change between 1.14.1 and 1.14.0? (Discussion #3132)


Did the application binary interface change between HDF5 1.14.0 and 1.14.1?

xref:
conda-forge/hdf5-feedstock#202 (comment)<https://github.com/conda-forge/hdf5-feedstock/pull/202#issuecomment-1592236319>
